webapp/topology.py

```diff
@@ -44,13 +44,16 @@ def get_tree(self) -> OrderedDict:
 
 class Site(object):
     # probably will have some other attributes like address, latitude, longitude, etc.
-    def __init__(self, name: str, id: int, facility: Facility):
+    def __init__(self, name: str, id: int, facility: Facility, site_info):
         self.name = name
         self.id = id
         self.facility = facility
+        self.other_data = site_info
+        if "ID" in self.other_data:
+            del self.other_data["ID"]
 
     def get_tree(self) -> OrderedDict:
-        return OrderedDict([("ID", self.id), ("Name", self.name)])
+        return OrderedDict([("ID", self.id), ("Name", self.name)] + list(self.other_data.items()))
```

Comment:
You should sort `list(self.other_data.items())` because currently the order is not defined.